custom_components/zwave_mqtt/discovery.py

```diff
@@ -298,6 +298,13 @@
             },
         ),
     },
+    {
+        const.DISC_COMPONENT: "sensor",
+        const.DISC_VALUES: dict(
+            DEFAULT_VALUES_SCHEMA,
+            **{const.DISC_PRIMARY: {const.DISC_COMMAND_CLASS: [CommandClass.BASIC]}},
```

Comment:
What's the difference between adding a new dict compared with just adding the CC to the sensor list above.

Reply:
The ValueGenre differs for the Basic CC so for the sake of readability I’ve put it in its own dict. I should have put some comment maybe as the discovery stuff is already hard to read/understand.